<launch>

<arg name="environment" default="sea"/>
<arg name="duration" default="30"/>
<arg name="feedback_method" default="attitude_with_depth"/>

<node pkg="bluerov_control_ros" type="bluerov_transect_node" name="bluerov_transect" output="screen">
        <param name="duration" value="$(arg duration)"/>
        <param name="environment" value="$(arg environment)"/>
        <param name="feedback_method" value="$(arg feedback_method)"/>

        <remap from="attitude_topic" to="/imu_tools/attitude/filtered"/>
        <remap from="pressure_topic" to="/mavros/imu/static_pressure"/>

    </node>
</launch>